core/templates/dev/head/components/summary_tile/CanonicalStorySummaryTileDirective.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for a canonical story tile.
+ */
+
+oppia.directive('canonicalStorySummaryTile', [
```

Comment:
Just a question, why canonicalStorySummaryTile? Is there any difference between canonical and additional story tiles (for when we do implement it)?

Reply:
Actually no. I shall change this

Reply:
Done